src/Graphics/Vty/Attributes.hs

```diff
@@ -58,7 +58,37 @@ module Graphics.Vty.Attributes
   , withURL
 
   -- * Colors
-  , module Graphics.Vty.Attributes.Color
```

Comment:
Would it be possible to revert this duplication of exports and just move the documentation to the `Color` module?

Reply:
Done.

I was initially hesitant to do this, since it requires rewriting a few functions; On second thought, it's obviously the best thing to do.